tests/unit/amo/components/TestLandingAddonsCard.js

```diff
@@ -75,6 +108,77 @@ describe(__filename, () => {
     expect(root.find(AddonsCard)).toHaveProp('footerLink', null);
   });
 
+  it('hides the footer link when there are less add-ons than LANDING_PAGE_THEME_COUNT', () => {
+    const addons = [];
+
+    for (let i = 0; i < LANDING_PAGE_THEME_COUNT - 1; i++) {
```

Comment:
You could probably use:

`const addons = Array(LANDING_PAGE_THEME_COUNT - 1).fill(createInternalAddon(fakeAddon));`

like is done in the `render` function at the top of the file, rather than manually pushing items onto an array.